src/Microsoft.VisualStudio.ProjectSystem.FSharp.VS/ProjectSystem/VS/Imaging/FSharpProjectImageProvider.cs

```diff
@@ -11,7 +11,6 @@ namespace Microsoft.VisualStudio.ProjectSystem.VS.Imaging
     /// </summary>
     [Export(typeof(IProjectImageProvider))]
     [AppliesTo(ProjectCapability.FSharp)]
-    [Order(2)]
```

Comment:
This is not needed anymore. Originally I had added the CSharp capability to F# projects as a quick way of getting everything to work and in that case, there were two providers providing icons and needed to be ordered. But now that I've removed that hack, there'll be just one and so we don't need this anymore.